charts/ingress/templates/ingress.yaml

```diff
@@ -15,6 +15,9 @@ metadata:
 {{- end }}
   name: {{ template "ingress.name" . }}
 spec:
+{{- if .Values.ingressClassName }}
+  ingressClassName: {{ .Values.ingressClassName }}
+{{- end -}}
```

Comment:
I don't think you want an - at the end of the end block. I think that is trimming your whitespace

Reply:
Good! Thanks for the review! I just deleted the - at the end of the end block!